### PR TITLE
added Cesium map; changed layout to match figma updated design 2

### PR DESCRIPTION
**Pull request by Anjali0407-git**

### CODE REVIEW

**Review by ankyAgrawal — APPROVED** [review]

**Comment by Anjali0407-git** [comment]
@ankyAgrawal Shall I merge it?

**Comment by Anjali0407-git** [comment]
@ankyAgrawal I am closing this PR without merging. As we discussed earlier, we will not merge frontend-updates into SADE-UI-main as of now. We will wait until the app is in good shape and stable to merge and then create the PR again.

**Pull request closed without merging**
